firmware/controllers/trigger/trigger_central.cpp

```diff
@@ -177,8 +177,12 @@ void hwHandleVvtCamSignal(trigger_value_e front) {
 }
 
 void hwHandleShaftSignal(trigger_event_e signal) {
-	if (!isUsefulSignal(signal, engineConfiguration)) {
-		return;
+	// for effective noise filtering, we need both signal edges, 
+	// so we pass them to handleShaftSignal() and defer this test
+	if (!boardConfiguration->useNoiselessTriggerDecoder) {
+		if (!isUsefulSignal(signal, engineConfiguration)) {
+			return;
+		}
```

Comment:
Note that the default behavior wasn't changed (useNoiselessTriggerDecoder==false)